charts/node/values.yaml

```diff
@@ -102,8 +102,11 @@ node:
   serviceMonitor:
     enabled: false
     #namespace: monitoring
-    #interval: 10s
-    # scrapeTimeout: 10s
+    interval: 1m
+    scrapeTimeout: 30s
+    targetLabels:
+      - chain
```

Comment:
if i understand correctly this label would conflict with the one from the directly exposed labels from the metrics.

Reply:
Oh yes, it's not good to have conflicts, then it would remap the existing `chain` label to `exported_chain` which is really confusing. See [prometheus configuration docs](https://prometheus.io/docs/prometheus/latest/configuration/configuration).

```
# [...] label conflicts are resolved by renaming
# conflicting labels in the scraped data to "exported_<original-label>" (for
# example "exported_instance", "exported_job") and then attaching server-side
# labels.
```

Reply:
In this case, it's probably better to only add `node` to the default `targetLabels`

Reply:
I removed the chain in the default values, and added `honor_labels` to avoid having conflict when a label is provided by he metrics.

```
If honor_labels is set to "true", label conflicts are resolved by keeping label
values from the scraped data and ignoring the conflicting server-side labels.
```